.github/workflows/scan-build.yml

```diff
@@ -13,24 +13,19 @@ jobs:
     steps:
       - name: Install Dependencies
         run: |
-          dnf -y install $COMPILER automake libtool pkgconf-pkg-config \
-              autoconf-archive git openssl-devel clang-analyzer
+          dnf -y install $COMPILER meson pkgconf-pkg-config \
+              git openssl-devel clang-analyzer
       - uses: actions/checkout@v3
         name: Checkout Repository
       - name: Setup
         run: |
-          autoreconf -fiv
-          ./configure
+          meson setup _build
       - name: Scan Build
         run: |
-          scan-build --html-title="PKCS#11 Provider ($GITHUB_SHA)" \
-                    --keep-cc \
-                    --status-bugs \
-                    --keep-going \
-                    -o scan-build.reports make
+          SCANBUILD=$PWD/.github/scan-build.sh ninja -C _build scan-build
```

Comment:
Why does this need to be spun out as an additional build script?

Reply:
This is the standard way of passing options to scan-build with Meson. See https://mesonbuild.com/howtox.html#use-clang-static-analyzer.